templates/layouts/_base.html

```diff
@@ -49,11 +49,14 @@
 
     <!-- Google Tag Manager -->
     <script nonce="{{ csp_nonce() }}">
+      var a = /^(.*)?\s*'usage':true\s*[^;]+(.*)?$/;
```

Comment:
Can not have a more meaningful variable name and add a comment about what this does?